src/common/utils.ts

```diff
@@ -9,21 +9,58 @@ import { Contributor } from "r2-shared-js/dist/es6-es2015/src/models/metadata-co
 import { IStringMap } from "r2-shared-js/dist/es6-es2015/src/models/metadata-multilang";
 import { diMainGet } from "readium-desktop/main/di";
 
+// https://github.com/IDPF/epub3-samples/blob/master/30/regime-anticancer-arabic/EPUB/package.opf
+//
+// "author": [
+//     {
+//         "name": {
+//             "ar": "دافيد  خيّاط لبروفيسورا"
+//         }
+//     },
+//     {
+//         "name": {
+//             "ar": "اردو هاتر ناتالي"
+//         }
+//     }
+// ],
+// "translator": {
+//     "name": {
+//         "ar": "فيّاض خليل مارينا"
+//     }
+// },
+// "contributor": {
+//     "name": "Vincent Gros",
+//     "role": "mrk"
+// },
+// "publisher": "Hachette Antoine",
+//
```

Comment:
Can you please use the latest code snippet? (sorry, I've just recently fixed bugs in the `r2-shared-js` parser):
```
        "author": [
            {
                "name": {
                    "ar": "دافيد  خيّاط لبروفيسورا",
                    "fr": "Pr David Khayat"
                }
            },
            {
                "name": {
                    "ar": "اردو هاتر ناتالي",
                    "fr": "Nathalie Hutter-Lardeau"
                }
            }
        ],
        "translator": {
            "name": {
                "ar": "فيّاض خليل مارينا",
                "fr": "Marina Khalil Fayad"
            }
        },
        "contributor": {
            "name": "Vincent Gros",
            "sortAs": "Gros, Vincent",
            "role": "mrk"
        },
        "publisher": "Hachette Antoine",
```

Thanks!